src/dfx/src/actors/replica.rs

```diff
@@ -102,8 +112,18 @@ impl Replica {
         }
     }
 
+    fn restart_replica_if_all_ready(&mut self, addr: Addr<Self>) {
+        let ready = !self.awaiting_canister_http_adapter_ready && !self.awaiting_btc_adapter_ready;
+        if ready {
```

Comment:
```suggestion
        let done_waiting = !self.awaiting_canister_http_adapter_ready && !self.awaiting_btc_adapter_ready;
        if done_waiting {
```

Can you reword something here? It took me 10 minutes until I realised that the line is not `let ready = !ready && !ready` but instead `let ready = !awaiting && !awaiting`. With all the talk about 'ready' my brain is too primed to understand everything as another form of 'ready'.